docs/alerts/webhook-connections/servicenow/index.md

```diff
@@ -6,48 +6,40 @@ description: With the ServiceNow integration, search results from Sumo Logic can
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-Your organization can use collected data to investigate issues across your deployment.
+Your org can use collected data to investigate issues across your deployment and then upload it to ServiceNow through the use of scheduled searches. After saving a search, results are available in ServiceNow. Additionally, you can launch ad hoc ServiceNow investigations using search results in Sumo Logic.
 
-Data is uploaded to ServiceNow via the use of scheduled searches. After saving a search, results are available in ServiceNow. Additionally, you can launch ad-hoc ServiceNow investigations using search results in Sumo Logic.
-
-To access [ServiceNow connections](/docs/alerts/webhook-connections/servicenow/), go to **Manage Data** > **Monitoring** > **Connections**.<br/><img src={useBaseUrl('img/connection-and-integration/connections.png')} alt="connections" />
-
-From here, you can also:
-* **Search Connections** to search. 
-* **Edit** Connections. 
-* **Delete** Connections. 
-* View the **Details** of Connections.
-
-* To create events in ServiceNow, see [Set Up ServiceNow Connections](/docs/alerts/webhook-connections/servicenow/set-up-connections).
-* To create incidents in ServiceNow, see [Set Up a ServiceNow Incident Webhook Connection](/docs/alerts/webhook-connections/servicenow/set-up-security-incident-webhook).
-* To import and maintain real-time information about AWS and VMware virtual machine assets into the ServiceNow CMDB, see the [Service Graph Installation and Configuration Guide](https://store.servicenow.com/appStoreAttachments.do?sys_id=d06467ab1bdab4d051a62132604bcb5e).
-* For the Sumo Logic Cloud SIEM integration with the ServiceNow Security Incident Response (SIR) module, see [Security Incident Response (SIR) Integration](../../../cse/integrations/security-incident-response-integration.md).
+To access [ServiceNow connections](/docs/alerts/webhook-connections/servicenow/), go to **Manage Data** > **Monitoring** > **Connections**. From here, you can search, edit, delete, and view of the details of Connections.
 
 In this section, we'll introduce the following concepts:
 
 <div className="box-wrapper">
 <div className="box smallbox card">
   <div className="container">
-  <a href="/docs/alerts/webhook-connections/servicenow/launch-investigations"><img src={useBaseUrl('img/icons/search.png')} alt="icon" width="40"/><h4>Launch Investigations</h4></a>
-  <p>Learn how to launch ad-hoc investigations directly from search results in Sumo Logic.</p>
+  <a href="/docs/alerts/webhook-connections/servicenow/set-up-security-incident-webhook"><img src={useBaseUrl('img/icons/security/security-capabilities.png')} alt="icon" width="40"/><h4>Set Up Security Incident Webhooks</h4></a>
```

Comment:
Move this card to the end.

Reply:
This was by far the top-ranked doc in Google Analytics for this section - I think I'll leave this here